### PR TITLE
fixes #6036

### LINKED ISSUE #6036: Improve test coverage for algorithms in load centrality
Currently we don't have full coverage for the algorithms in load centrality. Code blocks which are highlighted with red at codcov https://app.codecov.io/gh/networkx/networkx/blob/main/networkx/algorithms/centrality/load.py don't have corresponding tests. The tests should be added in https://github.com/networkx/networkx/blob/main/networkx/algorithms/centrality/tests/test_load_centrality.py

### Current Behavior

We don't test all the paths the code can take us.

### Expected Behavior

We should be testing everything so there aren't any surprises.

### Steps to Reproduce

Visit https://app.codecov.io/gh/networkx/networkx/blob/main/networkx/algorithms/centrality/load.py

### PR DESCRIPTION
**Pull request by Qudirah**

### CODE REVIEW

**Comment by MridulS** [comment]
Thanks for this @Qudirah!

Just a general note: Please make the title of the pull request a bit more detailed, it's hard to understand "fixes # some issue number" without searching for the issue. Something like "Improve test coverage for load centrality" can give reviewers a quick way of understanding the context of the changes in this pull request.

The commit message should also contain more information rather than just "fixed networkx#6036" as when someone is looking at the project after some time, 2-4 years after this is fixed they won't be able to quickly understand what does 6036 mean. So it's highly suggested to make commit messages and pull request titles as informative as possible :)

File: networkx/algorithms/centrality/tests/test_load_centrality.py [hunk 1]
```diff
@@ -41,6 +44,27 @@ def test_not_strongly_connected(self):
...
 
+    def test_P2_normalized_load(self):
+        G = self.P2
+        c = nx.load_centrality(G, normalized=True)
+        d = {0: 0.000, 1: 0.000}
+        for n in sorted(G):
+            assert c[n] == pytest.approx(d[n], abs=1e-3)
+
+    def test_C2_normalized_load(self):
+        G = self.C2
```
Comment (MridulS) [inline networkx/algorithms/centrality/tests/test_load_centrality.py:55]: This all looks good :)
But we don't really need to add `test_C2_normalized_load` and `test_K2_normalized_load`. Just one test is enough :)

Comment (MridulS) [reply networkx/algorithms/centrality/tests/test_load_centrality.py:55]: So could you remove them and just keep `test_P2_normalized_load`?

Thanks!

Comment (Qudirah) [reply networkx/algorithms/centrality/tests/test_load_centrality.py:55]: Okay! will solve this right now. Thanks for your guidance.

**Comment by MridulS** [comment]
Thanks for the contribution @Qudirah!

**Pull request merged**